NLog.Targets.Http/HTTP.cs

```diff
@@ -101,36 +93,48 @@ public bool IgnoreSslErrors
             }
         }
 
-        public bool FlushBeforeShutdown { get; set; } = true;
+        [Obsolete] public bool FlushBeforeShutdown { get; set; } = true;
```

Comment:
Why [Obsolete] ?

Reply:
Because it is not needed. NLog will automatically perform Flush before Shutdown,